src/components/Grid.jsx

```diff
@@ -414,8 +416,10 @@ export class Grid extends Component {
         stateful: this. props.stateful,
         showTreeRootNode: this.props.showTreeRootNode,
         menuState: this.props.menuState,
-        gridType: this.gridType
-    });
+        gridType: this.gridType,
+        rowRenderer: this.props.rowRenderer
```

Comment:
Hmm, probably don't want a default renderer. But this will go away anyway once `ROW` is added to `plugins`